nebuchadnezzar/nebu/h5p_injection.py

```diff
@@ -218,16 +229,22 @@ def _add_question(
             assert (
                 sub_library != "H5P.QuestionSet"
             ), "Question sets cannot contain question sets"
-            _add_question(i + 1, sub_library, sub_entry, questions)
+            _add_question(
+                [*id_parts, f"q-{i + 1}"], sub_library, sub_entry, questions
```

Comment:
Spelling out "question" might match with the use of `exercise_` earlier in the identifier and make it easier to find this line where the string is generated.

Also, why do underscore in `exercise_` but hyphen here?

Reply:
I was thinking that `_` was used to separate parts of the id and `-` was used to join segments within a part (like in a uuid, for example). I removed the hyphens in favor of adding the number directly to the end (i.e. question1 or answer1).